src/main.js

```diff
@@ -117,58 +114,73 @@ var quotes = [
   "Each person must live their life as a model for others.",
   "A champion is defined not by their wins but by how they can recover when they fall."
 ];
-
-var currentPoster = "";
-
+var savedPosters = [];
+var currentPoster = '';
 // event listeners go here 👇 revise with onClick etc
 window.addEventListener("load", function(){
   posterImage.src = getRandomIndex(images)
   posterTitles.innerText = getRandomIndex(titles);
   posterQuotes.innerText = getRandomIndex(quotes);
   showMyPoster.type = "button";
+  instantiateCurrentPoster();
 });
-
-  formButton.addEventListener("click", function(){
-    changePage(mainPage,formView)
-  });
- //.onClick
-  randomButton.addEventListener("click", function(){
-    posterImage.src = getRandomIndex(images);
-    posterTitles.innerText = getRandomIndex(titles);
-    posterQuotes.innerText = getRandomIndex(quotes);
+formButton.addEventListener("click", function(){  //.onClick
+  changePage(mainPage,formView)
+});
+randomButton.addEventListener("click", function(){
+  posterImage.src = getRandomIndex(images);
+  posterTitles.innerText = getRandomIndex(titles);
+  posterQuotes.innerText = getRandomIndex(quotes);
+  instantiateCurrentPoster();
```

Comment:
This now instantiates a new class when random button is clicked instead of on load event listener